coral-service/src/main/java/com/linkedin/coral/coralservice/utils/VisualizationUtils.java

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright 2022-2023 LinkedIn Corporation. All rights reserved.
+ * Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.coral.coralservice.utils;
+
+import java.io.File;
+import java.util.UUID;
+
+import org.apache.calcite.rel.RelNode;
+import org.apache.calcite.sql.SqlNode;
+
+import com.linkedin.coral.hive.hive2rel.HiveToRelConverter;
+import com.linkedin.coral.incremental.RelNodeIncrementalTransformer;
+import com.linkedin.coral.transformers.CoralRelToSqlNodeConverter;
+import com.linkedin.coral.trino.trino2rel.TrinoToRelConverter;
+import com.linkedin.coral.vis.VisualizationUtil;
+
+import static com.linkedin.coral.coralservice.utils.CoralProvider.*;
+import static com.linkedin.coral.coralservice.utils.RewriteType.*;
+
+
+public class VisualizationUtils {
+
+  public static File createImageDir() {
+    return new File(System.getProperty("java.io.tmpdir") + "/images" + UUID.randomUUID());
```

Comment:
When do we clean up? Is there a reason to keep previous query images after user moves on to another query on the frontend? Ignore caching.

Reply:
Some ideas: https://stackoverflow.com/questions/17933508/how-to-delete-a-file-after-sending-it-in-a-web-app